### PR TITLE
make documentation link more obvious

### PR DESCRIPTION
**Pull request by amirreza8002**
### Pull Request check-list

- [ x] Do tests and lints pass with this change?
- [ x] Do the CI tests pass with this change (enable it first in your forked repo and wait for the github action build to finish)?
- [ x] Is the new or changed code fully tested?
- [ x] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?
- [ x] Is there an example added to the examples folder (if applicable)?

### Description of change
added a link

### CODE REVIEW

**Comment by mkmkme** [comment]
Thanks for your contribution! The change looks fine, but could you please sign-off the commit you pushed?
https://github.com/valkey-io/valkey-py/pull/98/checks?check_run_id=30516332278

**Comment by amirreza8002** [comment]
hi
this is something i hadn't done before,
did the push fix it?

**Comment by mkmkme** [comment]
Yup, you got it right :)

File: README.md [hunk 1]
```diff
@@ -85,6 +85,9 @@ Alternatively, you might want to look at [Async connections](https://valkey-py.r
 
 There is built-in support for all of the [out-of-the-box Valkey commands](https://valkey.io/commands). They are exposed using the raw Redis command names (`HSET`, `HGETALL`, etc.) except where a word (i.e. del) is reserved by the language. The complete set of commands can be found [here](https://github.com/valkey-io/valkey-py/tree/main/valkey/commands), or [the documentation](https://valkey-py.readthedocs.io/en/latest/commands.html).
 
+## Documentation
+check out the [Documentation](https://valkey-py.readthedocs.io/en/latest/index.html)
```
Comment (aiven-sal) [inline README.md:89]: Just a little bit of nit-picking:
could you capitalize the first C and leave an empty line after `## Documentation`

**Comment by amirreza8002** [comment]
fair enough
i should work on my English literature :p

**Comment by mkmkme** [comment]
Now you forgot to sign the commit while fixing its contents :) Please sign it again just like the first time.

File: README.md (hunk 1 above)
Comment (mkmkme) [reply README.md]: I would also like the word "documentation" _in the link_ to be non-capitalized as it's in the middle of the sentense.

**Comment by amirreza8002** [comment]
that was too much problem for two lines of doc
i apologize

**Review by aiven-sal — APPROVED** [review]
Thank you very much and sorry for all the back and forth!

**Pull request merged**